stubs/plutus-ghc-stub/src/Plugins.hs

```diff
@@ -110,9 +109,7 @@ instance Semigroup PluginRecompile where
 
 instance Monoid PluginRecompile where
   mempty = NoForceRecompile
-#if __GLASGOW_HASKELL__ < 840
```

Comment:
Can you remind me what the GHC stub stuff is for?

Reply:
I also dnt know what the ghc stub is for.

I did this change since the number `840` points to ghc `8.40.X` which is not a ghc version ever released. Most likely the original author wanted to say `804` which is ghc `8.4.X`. Further, since we don't even support ghc<8.4 , I thought the check is redundant